README.md

```diff
@@ -31,18 +31,33 @@ Install dependencies for [`openai.embeddings_utils`](openai/embeddings_utils.py)
 pip install openai[embeddings]
 ```
 
+If you receive an error message stating: _no matches found: openai[embeddings]_, try running:
```

Comment:
I like the idea! This feels a bit redundant have it 3 times in the README. Can we consolidate it?